packages/grafana-data/src/datetime/common.ts

```diff
@@ -1,17 +1,46 @@
 import { TimeZone, DefaultTimeZone } from '../types/time';
 
+/**
+ * Type describing the date and time helper function options. Used for all the
+ * helper functions available to parse and format date and time values.
+ *
+ * @public
+ */
 export interface DateTimeOptions {
+  /**
+   * Specify if you want to override the timeZone used when parsing or formatting
+   * a date and time value. If no timeZone is set the default timeZone for the current
+   * user will be used.
+   */
   timeZone?: TimeZone;
 }
 
+/**
+ * Type to describe the time zone resolver function that will be used to access
+ * the default time zone of an user.
```

Comment:
```suggestion
 * the default time zone of a user.
```

Reply:
English is tricky here. The general rule is "an" if the next word starts with a vowel, but you need to go with the sound. "User" sounds like "yoo-zer," so that is a consonant sound. 
A user
A unicorn
An unclear rule with lots of loopholes. :)

Reply:
Thanks, I love that you are coaching me in this ❤️ 🙏